packages/common/src/LRUCache.ts

```diff
@@ -0,0 +1,32 @@
+export function getLRUCache<V>(max = 1000) {
```

Comment:
This should be out of scope here and be done within envelop 😇

Reply:
https://github.com/dotansimha/envelop/pull/1371